src/helpers/file.ts

```diff
@@ -1,4 +1,4 @@
 // We need a unique file name so we use span tags like the pipeline URL,
 // which can contain dots and other unsafe characters for filenames.
 // We filter them out here.
-export const getSafeFileName = (unsafeFileName: string) => unsafeFileName.replace(/[^a-z0-9]/gi, '_')
+export const getSafeFileName = (unsafeFileName: string) => unsafeFileName.replace('/', '\\')
```

Comment:
why this change? wouldn't it still be useful to get rid of any character not in `a-z0-9`?